test/TestProject/TestProject.csproj

```diff
@@ -9,7 +9,6 @@
     <ReadCommittedSnapshot>True</ReadCommittedSnapshot>
     <ServiceBrokerOption>EnableBroker</ServiceBrokerOption>
     <PackageProjectUrl>https://github.com/jmezach/MSBuild.Sdk.SqlProj/</PackageProjectUrl>
```

Comment:
`jmezach` should probably be replaced with `rr-wfm` here while we're at it ;).

Reply:
I have created an issue for this - happy to make a separate PR for it?

https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/issues/137

Reply:
Makes sense. And yes, a PR for this is more than welcome :).